.github/workflows/verify.yml

```diff
@@ -74,47 +74,38 @@ jobs:
             -DANDROID_ABI=arm64-v8a
             -DANDROID_PLATFORM=android-23
             -G Ninja
-        
+
         # - name: macOS
         #   os_identifier: mac
         #   os: macos-latest
         #   extra_flags: >
         #     -DCMAKE_C_COMPILER=clang
         #     -DCMAKE_CXX_COMPILER=clang++
         #     -DCMAKE_BUILD_TYPE=Debug
-          
+
     name: Test Offsets ${{ matrix.config.name }}
     runs-on: ${{ matrix.config.os }}
 
     steps:
-    - uses: actions/checkout@v4
+    - name: Checkout bindings
+      uses: actions/checkout@v4
 
-    - uses: actions/checkout@v4
+    - name: Checkout geode
+      uses: actions/checkout@v4
       with:
         repository: geode-sdk/geode
         path: geode
+        # TODO: remove on merge
+        ref: codegen-on-configure
 
     - name: Set up codegen binary cache
       id: codegen-cache
       uses: actions/cache@v3
       with:
-        path: build/codegen
+        path: build/bindings/codegen
         # cache by os since for android32 and 64 the codegen binary is the same
         key: codegen-${{ hashFiles('codegen/**') }}-${{ matrix.config.os }}
 
-    - name: Set CPM_SOURCE_CACHE
-      shell: bash
-      run: echo "CPM_SOURCE_CACHE=${{ github.workspace }}/cpm-cache" >> $GITHUB_ENV
-    
-    - name: Set up CPM cache
-      id: cache-cpm
-      uses: actions/cache@v3
-      with:
-        path: ${{ github.workspace }}/cpm-cache
-        key: cpm-cache-${{ hashFiles('**/CMakeLists.txt', '**/*.cmake') }}
-        restore-keys: |
-          cpm-cache-
```

Comment:
my cache :(

Reply:
not needed since member test doesnt use any cpm packages anymore